deploy/cloud-foundry/db-migration/db-migrate.sh

```diff
@@ -23,7 +23,12 @@ case $DB_TYPE in
     echo "Migrating postgresql instance on $DB_HOST"
     $GOBIN/goose -env cf_postgres up
     if [ $? -eq 0 ]; then
-        while sleep 60; do echo "Database successfully migrated. Please restart the application via 'cf push -c \"null\"'"; done
+        while
```

Comment:
I think its more common to put:

while true; do
    echo "Database successfully migrated. Please restart the application via 'cf push -c \"null\"'";
    sleep 60
done